mscore/stafftextproperties.cpp

```diff
@@ -39,7 +39,7 @@ static void initChannelCombo(QComboBox* cb, StaffTextBase* st)
       Fraction tick = static_cast<Segment*>(st->parent())->tick();
       for (const Channel* a : part->instrument(tick)->channel()) {
             if (a->name().isEmpty() || a->name() == Channel::DEFAULT_NAME)
-                  cb->addItem(QObject::tr(Channel::DEFAULT_NAME));
```

Comment:
this never worked apparently, that string ('normal', prior to this PR) never made it to Transifex